docs/config.md

```diff
@@ -276,6 +276,14 @@ Some ready-to-use trailer templates are available for frequently used trailers:
 
 Existing trailers are also accessible via `commit.trailers()`.
 
+The recognizable trailer separators can be customized with the
+`trailer.separators` configuration. Its default value is `:`.
```

Comment:
Same as above comment. You might also want to link to <https://git-scm.com/docs/git-interpret-trailers#Documentation/git-interpret-trailers.txt-trailerseparators>